core/bind/core_bind.cpp

```diff
@@ -1673,8 +1673,14 @@ int _Directory::get_current_drive() {
 }
 
 Error _Directory::change_dir(String p_dir) {
-	ERR_FAIL_COND_V_MSG(!is_open(), ERR_UNCONFIGURED, "Directory must be opened before use.");
-	return d->change_dir(p_dir);
+	Error err = d->change_dir(p_dir);
+	
```

Comment:
Trailing whitespace here and on line 1682.

I suggest setting up the pre-commit hooks locally to run clang-format when you commit: https://github.com/godotengine/godot/tree/master/misc/hooks